server/app/dal/CassandraCluster.scala

```diff
@@ -27,25 +27,34 @@ class CassandraCluster @Inject()(conf: CassandraConfig, appLifecycle: Applicatio
 }
 
 @Singleton
-class CassandraConfig @Inject()(configuration: Configuration) {
+class CassandraConfig @Inject()(configuration: Configuration, environment: Environment) {
   val config: Config = configuration.underlying
 
   val keySpace = config.getString("devgym.db.cassandra.keyspace")
   val port = config.getInt("devgym.db.cassandra.port")
 
   lazy val hosts: Seq[String] = {
+    val hostsConf = "devgym.db.cassandra.hosts"
+    val dockerConf = "devgym.db.cassandra.docker"
+
     def hosts: Seq[String] = {
       import scala.collection.JavaConversions._
-      config.getStringList("devgym.db.cassandra.hosts")
+      config.getStringList(hostsConf)
     }
 
-    def docker: Option[Boolean] = configuration.getBoolean("devgym.db.cassandra.docker")
+    def docker: Option[Boolean] = configuration.getBoolean(dockerConf)
 
-    def ip() = docker match {
-      case Some(true) => Try(Seq("docker-machine", "ip", "default").!!.trim).toOption
+    def dockerIp() = docker match {
+      case Some(true) =>
+        if (environment.mode == Mode.Prod)
```

Comment:
It is usually bad practice when application code has some distinction related to the environment. There is a check that current environment is PROD.